src/techs/techs.service.ts

```diff
@@ -124,8 +131,14 @@ export class TechsService {
         //check for valid teamId
         await this.validateTeamId(teamId);
 
+        //check if user is a member of the team
         manageOwnVoyageTeamWithIdParam(req.user, teamId);
```

Comment:
I think we won't need `manageOwnVoyageTeamWithIdParam` if we have `userCanChangeCategory`
Also could we move `userCanChangeCategory` to the `ability` folder?

Reply:
I tried removing this, but got a testing error.  Looking at it closer, these functions are comparing different values.  `manageOwnVoyageTeamWithIdParam` verifies that user is in the team from the URI params, and `userCanChangeCategory` verifies that the user is in the team from `category.voyageTeamId`

Reply:
but I mean if the user is in the team in `category.voyageTeamId` then we can assume they are allowed to update it, without having to check if the user is in the team from the uri params? I guess we only need to check the one which is more restrictive

Reply:
Ok - took it out and fixed the test